packages/docs/docs/medications/formulary.md

```diff
@@ -0,0 +1,298 @@
+---
+keywords:
+  - drugs
+  - formulary
+  - medication
+---
+
+# Modeling a Formulary
+
+## Introduction
+
+A formulary is a catalog of drugs offered by your organization. When implementing a custom EMR, you may want to include essential metadata about these drugs to help prescribing physicians, pharmacists, and patients.
+
+The principal resource for building a digital formulary is the [`MedicationKnowledge`](/docs/api/fhir/resources/medicationknowledge) resource. This guide aims to explore the most pertinent attributes of [`MedicationKnowledge`](/docs/api/fhir/resources/medicationknowledge) for digital health providers.
+
+This guide is informed by the [DaVinci Payer Data Exchange (PDex) US Drug Formulary](https://build.fhir.org/ig/HL7/davinci-pdex-formulary/index.html) implementation guide. [The Da Vinci Project](http://www.hl7.org/about/davinci/index.cfm) includes industry leaders, including Humana, Cigna, and Optum, who are using FHIR to support value-based care. You can check out how Humana implements this API [here](https://developers.humana.com/apis/drug-formulary-api/doc).
+
+## Drug Code
+
+When defining `MedicationKnowledge.code`, it's highly recommended to use RxNorm as the primary coding system. The Da Vinci guide distinguishes two different types of codes: "semantic drugs" (mandatory) and "semantic drug form group" (optional).
+
+- "Semantic drugs" equate to the RxNorm term types of `Semantic Clinical Drug (SCD)`, `Semantic Branded Drug (SBD)`, `Generic Pack (GPCK)`, or `Branded Pack (BPCK)`,
+- "Semantic drug form group" matches the term types of `Semantic Clinical Drug Form (SCDG)` and `Semantic Branded Drug Form Group (SBDG)`.
+
+For a deeper understanding of RxNorm, refer to our [RxNorm guide](./medication-codes).
+
+## Drug Characteristics
+
+Details about the physical characteristics of a drug, such as its color, shape, size, and imprint, can be found in the `drugCharacteristics` section of the [`MedicationKnowledge`](/docs/api/fhir/resources/medicationknowledge) resource. The [Medication knowledge characteristic code](http://hl7.org/fhir/R4/valueset-medicationknowledge-characteristic.html) valueset can be a handy reference for examples of `MedicationKnowledge.drugCharacteristics.type`.
+
+Other metadata fields to note include `packaging`, `doseForm`, and `intendedRoute`.
+
+- The `packaging` attribute represents the type of packaging and quantity of the drug in the package. For instance, `'{tbl}'` is used to indicate a tablet. More details can be found in the [MedicationKnowledge Package Type Codes](http://hl7.org/fhir/R4/valueset-medicationknowledge-package-type.html) and the guide to [units of measure](https://terminology.hl7.org/4.0.0/ValueSet-v3-UnitsOfMeasureCaseSensitive.html).
```

Comment:
`tbl` does not appear in this doc http://hl7.org/fhir/R4/valueset-medicationknowledge-package-type.html

Reply:
Good catch - this was a chatgpt-ism I thought I had edited out